frontend/exporter/src/lib.rs

```diff
@@ -9,33 +7,44 @@
 #![allow(incomplete_features)]
 #![feature(specialization)]
 #![allow(rustdoc::private_intra_doc_links)]
+#![cfg_attr(feature = "rustc", feature(rustc_private))]
 
-extern crate rustc_abi;
-extern crate rustc_ast;
-extern crate rustc_ast_pretty;
-extern crate rustc_data_structures;
-extern crate rustc_driver;
-extern crate rustc_errors;
-extern crate rustc_hir;
-extern crate rustc_hir_analysis;
-extern crate rustc_index;
-extern crate rustc_infer;
-extern crate rustc_interface;
-extern crate rustc_middle;
-extern crate rustc_mir_build;
-extern crate rustc_session;
-extern crate rustc_span;
-extern crate rustc_target;
-extern crate rustc_trait_selection;
-extern crate rustc_type_ir;
+cfg_if::cfg_if! {
```

Comment:
We don't need `cfg_if` for this. Just define a macro in here. See for example what we do in [libcrux](https://github.com/cryspen/libcrux/blob/main/libcrux-ml-kem/src/lib.rs#L177).